cmake/faceliftMacros.cmake

```diff
@@ -359,6 +361,8 @@ macro(_facelift_parse_target_arguments additionalOptions additionalOneValueArgs
     set(oneValueArgs ${additionalOneValueArgs})
     set(multiValueArgs PRIVATE_DEFINITIONS PUBLIC_DEFINITIONS PROPERTIES COMPILE_OPTIONS
         HEADERS HEADERS_GLOB HEADERS_GLOB_RECURSE
+        HEADERS_NO_MOC
+        HEADERS_NO_MOC_FILE
```

Comment:
HEADERS_NO_MOC_FILE is one value arg, should go there

> not needed

It is needed in our case AFAIK. To be confirmed yet.